site/docs/components/overview.mdx

```diff
@@ -61,6 +61,10 @@ It is common for components to come with default margins surrounding them and ye
 Use "Small", "Medium", "Large": in code, `sm`, `md`, `lg`. For more information, see [Sizing guidelines](https://cultureamp.design/guidelines/sizing).
 
 
+# See also
+
+- [Component health](/component_health) for the statuses of all Kaizen components at a glance.
```

Comment:
```suggestion
- [Component health](/component-health) for the statuses of all Kaizen components at a glance.
```

Small one, noticed this hasn't been updated to the new URL

Reply:
🤦 I checked that this rendered correctly but forgot to click it, nice catch